source/api_documentation/transactions_api/optional_fields.txt

```diff
@@ -41,43 +41,43 @@
     - URL to the call recording, if available.
 
   * - signal_1_value
-    - Signal: Credit Card
+    - Voice Signal: Credit Card
     - Signals are configurable metrics that are tracked across calls, configured with rules about when the Signal is met (1 if Signal is met, 0 if Signal was applied to the call but not met, null if Signal was not applied to call)
```

Comment:
Should we say "Voice signals are configurable ..... and when the Voice signal is met (1 if Voice signal is met) ..." etc?  To drive home that the 1, 0, null is for voice signals only.   

Since further below our `signal_value` (which is for "outcomes") is true/false as its value.